modules/reference/pages/systems/system-details/sd-configuration.adoc

```diff
@@ -1,154 +1,74 @@
-Also, locally-managed configuration files are a method by which system group administrators who do not have configuration administration privileges can manage configuration files on the machines they can manage.
-+
-image::system_details_traditional_configuration_view_mod_local_paths.png[scaledwidth=80%]
 
-Local Sandbox [Management]::
-In the sandbox you can store configuration files under development.
-You can promote files from the sandbox to a centrally-managed configuration channel using [guimenu]``Copy Latest to Central Channel``.
-After files in this sandbox have been promoted to a centrally-managed configuration channel, you can deploy them to other systems.
-+
-Use [guimenu]``Copy Latest to System Channel`` to install a configuration on the local system only.
-When done, the file will end up on the [guimenu]``Locally-Managed Files`` subtab.
-+
-image::system_details_traditional_configuration_view_mod_sandbox_paths.png[scaledwidth=80%]
-
-
-
-[[sd-config-add-files]]
-== Add Files
-
-To upload, import, or create new configuration files, open the [guimenu]``Add Files`` subtab.
-
-Upload File::
-To upload a configuration file from your local machine, browse for the upload file, specify whether it is a text or binary file, enter [guimenu]``Filename/Path`` and user and group ownership.
-Specific file permissions can be set.
-When done, click btn:[Upload Configuration File].
-
-image::system_details_traditional_configuration_add_files_upload.png[scaledwidth=80%]
-
-Import Files::
-Via the [guimenu]``Import Files`` tab, you can add files from the system you have selected before and add it to the sandbox of this system.
-Files will be imported the next time [command]``mgr_check`` runs on the system.
-To deploy these files or override configuration files in global channels, copy this file into your local override channel after the import has occurred.
-
-In the text box under [guimenu]``Import New Files`` enter the full path of any files you want import into {productname} or select deployable configuration files from the [guimenu]``Import Existing Files`` list.
-When done, click btn:[Import Configuration Files].
-
-image::system_details_traditional_configuration_add_files_import.png[scaledwidth=80%]
-
-Create File::
-Under [guimenu]``Create File``, you can directly create the configuration file from scratch.
-Select the file type, specify the path and file name, where to store the file, plus the symbolic link target file name and path.
-Ownership and permissions and macro delimiters need to be set.
-For more information on using macros, see xref:reference:configuration/files-locally-managed.adoc#s3-sm-file-macros[].
-
-In the [guimenu]``File Contents`` text box, type the configuration file.
-Select the type of file you are creating from the drop-down box.
-Possible choices are Shell, Perl, Python, Ruby and XML.
-When done, click btn:[Create Configuration File].
-
-
-
-[[sd-config-deploy-files]]
```

Comment:
Not useful anymore? :thinking:

Reply:
Bits have been rewritten, other bits have been moved. Feel free to rescue anything that you feel is missing though.